CONP_DATS_fields.md

```diff
@@ -13,7 +13,7 @@ We use the `DATS.json` file format to store metadata describing CONP datasets.
 |Formats|distributions - format|**REQUIRED** The technical format of the dataset distribution. Use the file extension or MIME type when possible. (Definition adapted from DataCite)Primary data files format (*example*: csv, nifti, txt, fasta).| The technical format of the dataset distribution. Use the file extension or MIME type when possible. (Definition adapted from DataCite).|
 |Size|distributions - size|**REQUIRED** Total size of the dataset.| The quantity of space required on disk (or other medium) for this dataset.|
 |Units|distributions - unit|**REQUIRED** Unit in which the size is measured.(*example*: GB)|Unit in which the size is measured.(KB -> KiloByte, MB -> MegaByte, GB -> GigaByte, TB -> TeraByte and PB -> PetaByte). |
-|Landing Page|distributions - access - landingPage|**REQUIRED** Web address (URL) where the original dataset can be found.|A URL (Web page) that contains information about the associated dataset (with a link to the dataset) or a direct link to the dataset itself. |
+|Landing Page|distributions - access - landingPage|**REQUIRED** Web address (URL) where the original dataset can be found. Default value points to the README.md for the dataset. |A URL (Web page) that contains information about the associated dataset (with a link to the dataset) or a direct link to the dataset itself. |
```

Comment:
I wonder if we should also update the help text for the GUI here or do you think it would confuse people? I am on the fence...

Reply:
It might be a good idea to have the GUI autofill this unless the user specifies a value; we definitely want to update the help text for the GUI, but what we update it to will differ based on whether we do that; I will make that an issue and assign it to @liamocn.

Reply:
Done: https://github.com/CONP-PCNO/conp-portal/issues/370